src/components/form/DatePickerField.svelte

```diff
@@ -29,33 +31,51 @@
   }
 </script>
 
-<div class="date-picker-field">
-  <Input {layout}>
-    {#if label}
-      {#if layout === 'inline'}
-        <label use:tooltip={{ content: label, placement: 'top' }} class:error={$field.invalid} for={name}>{label}</label
-        >
-      {:else}
-        <label class:error={$field.invalid} for={name}>{label}</label>
+{#if !useFallback}
+  <div class="date-picker-field">
+    <Input {layout}>
+      {#if label}
+        {#if layout === 'inline'}
+          <label use:tooltip={{ content: label, placement: 'top' }} class:error={$field.invalid} for={name}
```

Comment:
I think this has carried over the weird HTML formatting with the closing brackets for tags being placed on the next line. Do you mind fixing that here since it's already been modified?

Reply:
Good catch